pkg/apis/eventsource/v1alpha1/types.go

```diff
@@ -714,6 +714,9 @@ type GithubEventSource struct {
 	// Repositories holds the information of repositories, which uses repo owner as the key,
 	// and list of repo names as the value
 	Repositories []OwnedRepositories `json:"repositories,omitempty" protobuf:"bytes,15,rep,name=repositories"`
+	// Whether to skip TLS verification when connecting to GitHub for webhook creation, useful for GitHub Enterprise use case.
+	// +optional
+	GithubSkipVerifyTLS bool `json:"githubSkipVerifyTLS,omitempty" protobuf:"varint,16,opt,name=githubSkipVerifyTLS"`
```

Comment:
Could we just. use `SkipVerifyTLS`? It sounds redundant since the spec is already under a field named `github`.

Reply:
This struct already has `Insecure` which refers to the target GitHub Webhook configuration.

I through that GitHubSkipVerifyTLS would ease the understanding of which field does what. What do you think?

Reply:
Can we do `APISkipVerifyTLS` instead, considering we have `APIToken` as well.

```yaml
spec:
  github:
    example:
      apiToken: xxxx
      apiSkipTLSVerify: true
```